lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1098,6 +1098,15 @@
     @include padding($baseline $baseline 0 $baseline);
     border: 1px solid $gray-l5;
   }
+
+  .padded-content {
```

Comment:
While I know we're doing a quick fix here, I'd prefer not to add these types of utility classes in 1) in general and 2) specifically into view-based Sass partials. Any chance you can use a different context for this?